charts/tenantfilesystem/templates/environment.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: apiextensions.crossplane.io/v1alpha1
```

Comment:
So you mean that we should reference to a global EnvironmentConfig ?
In that case, we need a default name or similar.
Because this setting is hardcoded.